src/ssl.c

```diff
@@ -13555,6 +13602,11 @@ int wolfSSL_GetSessionFromCache(WOLFSSL* ssl, WOLFSSL_SESSION* output)
 #else
     byte*        tmpTicket = NULL;
 #endif
+#ifdef WOLFSSL_TLS13
```

Comment:
`#if defined(HAVE_SESSION_TICKET) && defined(WOLFSSL_TLS13)`

Reply:
we are already in a region of code gated by HAVE_SESSION_TICKET